sdk/JavaScript/TestFramework/js/tests/pushTests.js

```diff
@@ -392,4 +281,27 @@ function definePushTestsNamespace() {
     };
 }
 
+function getRegistrations(channelUri, platform) {
```

Comment:
oh you are using internals, well I guess that makes sense, something to think on for next time we make modules where we wrap our code.